### PR TITLE
Change provenance format to Statement to avoid empty sigs.

### PR DESCRIPTION
**Pull request by msuozzo**

### CODE REVIEW

File: create_provenance.go [hunk 1]
```diff
@@ -222,22 +221,9 @@ func main() {
...
-	stmtCompactPayload, _ := json.Marshal(stmt)
-	envelope := Envelope{
-		PayloadType: PayloadContentType,
-		Payload:     base64.StdEncoding.EncodeToString(stmtCompactPayload),
-		Signatures:  []interface{}{},
-	}
-	envelopePayload, _ := json.MarshalIndent(envelope, "", "  ")
-	fmt.Println("Provenance:\n" + string(envelopePayload))
-	if err := ioutil.WriteFile(*outputPath, envelopePayload, 0755); err != nil {
+	payload, _ := json.MarshalIndent(stmt, "", "  ")
```
Comment (TomHennen) [inline create_provenance.go:224]: Is it worth a comment that says why we're just writing the statement and that at L2+ it will need to be signed?

**Review by TomHennen — CHANGES_REQUESTED** [review]

**Review by MarkLodato — APPROVED** [review]
Thanks, Matt!

File: .github/workflows/example-publish.yml [hunk 2]
```diff
@@ -37,4 +37,4 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: build-provenance
```
Comment (loosebazooka) [inline .github/workflows/example-publish.yml:39]: should we remove this name field from both?

Comment (msuozzo) [reply .github/workflows/example-publish.yml:39]: Not sure. We could so both the artifact and its provenance would be in the same archive. I'd be fine with either but regardless, it should be in a follow-on change.

File: create_provenance.go (hunk 1 above)
Comment (msuozzo) [reply create_provenance.go:227]: Done.

**Review by TomHennen — APPROVED** [review]

**Pull request merged**